var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -61,19 +61,30 @@ def autoreconf(self, spec, prefix):
         aclocal('-I', 'lib/autoconf')
         autoconf()
 
+    @property
+    def optflags(self):
+        flags = '-O2'
+
+        # Optimizations for the Intel compiler, suggested by CP2K
+        if '%intel' in self.spec:
+            flags += ' -xAVX -axCORE-AVX2 -ipo'
```

Comment:
I would probably just remove these. These are host-specific optimization flags. Until Spack has some way of knowing what host you are installing for (or cross-compiling for), it should be up to users to add host-specific flags to their `compilers.yaml`.

P.S. I think I'm actually the one who wrote this package and added this line. But that might have been before you could add compiler flags to `compilers.yaml`.